src/client/components/pages/Courses/MeetingModal.tsx

```diff
@@ -157,7 +157,7 @@ const MeetingModal: FunctionComponent<MeetingModalProps> = function ({
                     </h4>
                     <p>
                       {
-                        instructor.notes === ''
+                        !instructor.notes || instructor.notes === ''
```

Comment:
```suggestion
                        !instructor.notes
```

Since JavaScript considers `''` and `null` to both be a falsy values, you could probably get away with just the left side of the expression here.
![image](https://user-images.githubusercontent.com/50675045/117015817-14f17980-acc0-11eb-9afd-6faa9ff05917.png)

Reply:
Ah my bad! Thanks Robert.